### PR TITLE
Fixed access to element in size tensor for scripting

### PR DESCRIPTION
**Pull request by bpstark**
when using scripting, there was an error in attempting to access a
specific element from within the size tensor.

### CODE REVIEW

**Comment by kostmo** [comment]
## :pill: CircleCI build failures summary and remediations
As of commit a62c2270:


* **1/1** failures introduced in this PR


## Detailed failure analysis

One may explore the probable reasons each build failed interactively [on the Dr. CI website](https://dr.pytorch.org/commit-details.html?sha1=a62c22707c85a0464900f6a626d66c5bd36ae49a).

### :detective: 1 new failure recognized by patterns
The following build failures do not appear to be due to upstream breakage:
#### [See CircleCI build](https://circleci.com/gh/pytorch/pytorch/4348534) pytorch_macos_10_13_py3_test (1/1)
**Step:** "Test" ([full log](https://dr.pytorch.org/api/view-log-full?build_id=91365300) | [pattern match details](https://dr.pytorch.org/build-details.html?build_id=91365300))
<details>
<summary>
<code>Jan 27 10:48:35 RuntimeError: test_autograd failed!</code>
</summary>

```
Jan 27 10:48:35 Ran 1047 tests in 275.709s 
Jan 27 10:48:35  
Jan 27 10:48:35 FAILED (errors=1, skipped=13, expected failures=1) 
Jan 27 10:48:35  
Jan 27 10:48:35 Generating XML reports... 
Jan 27 10:48:35 Traceback (most recent call last): 
Jan 27 10:48:35   File "test/run_test.py", line 456, in <module> 
Jan 27 10:48:35     main() 
Jan 27 10:48:35   File "test/run_test.py", line 449, in main 
Jan 27 10:48:35     raise RuntimeError(message) 
Jan 27 10:48:35 RuntimeError: test_autograd failed! 
Jan 27 10:48:35 + cleanup 
Jan 27 10:48:35 + retcode=1 
Jan 27 10:48:35 + set +x 
```

</details>



---

<details><summary>This comment was automatically generated by <a href="https://github.com/kostmo/circleci-failure-tracker/tree/master/docs/from-pull-request-comment">Dr. CI</a> (expand for details).</summary>Follow <a href="https://dr.pytorch.org/admin/comments-opt-out.html">this link to opt-out</a> of these comments for your Pull Requests.

Please report bugs/suggestions on the <a href="https://github.com/kostmo/circleci-failure-tracker/issues">GitHub issue tracker</a>.
</details>

**Comment by bpstark** [comment]
Rebased the branch and added a test for scripting specifically.

**Comment by yf225** [comment]
@houseroad Curious do you know who should review it? Thanks!

**Comment by houseroad** [comment]
@yf225 usually, I rely on @spandantiwari, @lara-hdr and @BowenBao to do the review first.  Before landing, I usually will take a look to make sure the changes won't break our internal part.

File: torch/onnx/symbolic_opset11.py
Comment (BowenBao) [inline torch/onnx/symbolic_opset11.py]: nit: can we import and use `__getitem_` in opset 9?

**Review by BowenBao — APPROVED** [review]
LGTM, minor comment on code reuse.

**Review by lara-hdr — APPROVED** [review]

File: torch/onnx/symbolic_opset11.py
Comment (bpstark) [reply torch/onnx/symbolic_opset11.py]: done

**Pull request closed without merging**
